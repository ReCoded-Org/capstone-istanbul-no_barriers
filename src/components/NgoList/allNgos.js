import image from "./image/image.jpg";

// This will be a fetch from the database
const allNgos = [
  {
    id: "firebaseDocIda",
    image,
    name: "Teng",
    service: "Health Care",
    city: "Ankara",
    rating: "4",
    details: {
      facilityName: "Teng clinic",
      address: "Çankaya/Ankara",
      serviceDescription:
        "Suspendisse pulvinar pretium scelerisque. Suspendisse potenti. Sed ullamcorper enim sed lacinia sagittis. Nulla tellus risus, maximus et vestibulum id, tempor eget ipsum. Praesent convallis dictum turpis eu ullamcorper. Curabitur in ultrices sapien. Etiam lacinia ante quis luctus luctus. Nullam efficitur turpis orci, at bibendum tortor efficitur sed. Nam accumsan dolor luctus blandit lobortis.",
      responsiblePerson: "Ali Demir",
      phoneNumber: "+90 000 000 00",
      email: "tengAdmin@teng.com",
      website: "www.teng.com",
    },
  },
  {
    id: "firebaseDocIdb",
    image,
    name: "Teng",
    service: "Health Care",
    city: "Ankara",
    rating: "4",
    details: {
      facilityName: "Teng clinic",
      address: "Çankaya/Ankara",
      serviceDescription:
        "Suspendisse pulvinar pretium scelerisque. Suspendisse potenti. Sed ullamcorper enim sed lacinia sagittis. Nulla tellus risus, maximus et vestibulum id, tempor eget ipsum. Praesent convallis dictum turpis eu ullamcorper. Curabitur in ultrices sapien. Etiam lacinia ante quis luctus luctus. Nullam efficitur turpis orci, at bibendum tortor efficitur sed. Nam accumsan dolor luctus blandit lobortis.",
      responsiblePerson: "Ali Demir",
      phoneNumber: "+90 000 000 00",
      email: "tengAdmin@teng.com",
      website: "www.teng.com",
    },
  },
  {
    id: "firebaseDocIdc",
    image,
    name: "ABC",
    service: "Education",
    city: "Şanlıurfa",
    rating: "4",
    details: {
      facilityName: "ABC School",
      address: "Şanlıurfa",
      serviceDescription:
        "Suspendisse pulvinar pretium scelerisque. Suspendisse potenti. Sed ullamcorper enim sed lacinia sagittis. Nulla tellus risus, maximus et vestibulum id, tempor eget ipsum. Praesent convallis dictum turpis eu ullamcorper. Curabitur in ultrices sapien. Etiam lacinia ante quis luctus luctus. Nullam efficitur turpis orci, at bibendum tortor efficitur sed. Nam accumsan dolor luctus blandit lobortis.",
      responsiblePerson: "Aişe Şen",
      phoneNumber: "+90 000 000 00",
      email: "abcAdmin@abc.com",
      website: "www.abc.com",
    },
  },
];

export default allNgos;
