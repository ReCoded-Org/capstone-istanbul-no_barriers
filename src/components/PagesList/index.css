:root {
    --main-ngoTable-bg-color: #eff4f6;
}
