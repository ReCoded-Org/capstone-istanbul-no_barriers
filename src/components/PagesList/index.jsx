import React from 'react';

const PagesList = () => {
    return (
        <div>
            
        </div>
    );
}

export default PagesList;
