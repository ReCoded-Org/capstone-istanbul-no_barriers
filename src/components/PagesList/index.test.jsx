import React from "react";
import renderer from "react-test-renderer";
import PagesList from "./index";
import allNgos from "../NgoList/allNgos";

it("PagesList renders correctly", () => {
  const tree = renderer.create(<PagesList ngos={allNgos} />).toJSON();
  expect(tree).toMatchSnapshot();
});
