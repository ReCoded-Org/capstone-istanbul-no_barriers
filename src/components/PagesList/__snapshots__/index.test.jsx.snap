// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagesList renders correctly 1`] = `
<div
  className="pages"
>
  <ul
    className="d-flex justify-content-end pb-5 pagination"
    style={
      Object {
        "backgroundColor": "#eff4f6",
        "marginBottom": "0",
        "paddingRight": "6rem",
        "paddingTop": "1rem",
      }
    }
  >
    <li
      className="page-item"
    >
      <a
        className="page-link"
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
      >
        <span
          aria-hidden="true"
        >
          «
        </span>
        <span
          className="sr-only"
        >
          First
        </span>
      </a>
    </li>
    <li
      className="page-item"
    >
      <a
        className="page-link"
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
      >
        1
      </a>
    </li>
    <li
      className="page-item"
    >
      <a
        className="page-link"
        href="#"
        onClick={[Function]}
        onKeyDown={[Function]}
        role="button"
      >
        <span
          aria-hidden="true"
        >
          »
        </span>
        <span
          className="sr-only"
        >
          Last
        </span>
      </a>
    </li>
  </ul>
</div>
`;
